src/main/java/frc/robot/VisionFieldLayoutRecognizer.java

```diff
@@ -0,0 +1,120 @@
+package frc.robot;
+
+import java.util.List;
+import java.util.ArrayList;
+
+ public class VisionFieldLayoutRecognizer
+    implements PixyFieldPoseResolverInterface{
+
+    private List<Double> distanceRanges;
+    
+    public VisionFieldLayoutRecognizer() {
+      DistanceRange();
+    }
+
+    public void DistanceRange(){
```

Comment:
method names start with lower case letters

Reply:
this method is horribly named. its name should say what you're doing. like, setupDistanceRanges ( emphasis on the setup)